includes/definitions/cdata.yaml

```diff
@@ -10,5 +10,8 @@ over:
     - { graph: device_processor, text: 'CPU Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
 discovery:
-    - sysObjectID:
-        - .1.3.6.1.4.1.17409
+    - 
+        sysObjectID:
+            - .1.3.6.1.4.1.17409
+        sysObjectID_except:
+            - .1.3.6.1.4.1.17409.1
```

Comment:
regression added by #10975 (which was not detected by Travis in #10975 for some reason)